src/GraphQLParser.Tests/GraphQLAstVisitorTests.cs

```diff
@@ -54,8 +54,8 @@ public GraphQLAstVisitorTests()
 
         [Theory]
         [InlineData(IgnoreOptions.None)]
-        [InlineData(IgnoreOptions.IgnoreComments)]
-        [InlineData(IgnoreOptions.IgnoreCommentsAndLocations)]
+        [InlineData(IgnoreOptions.Comments)]
+        [InlineData(IgnoreOptions.All)]
```

Comment:
```suggestion
        [InlineData(IgnoreOptions.Comments)]
        [InlineData(IgnoreOptions.Locations)]
        [InlineData(IgnoreOptions.All)]
```
For the sake of consistency, shouldn't Locations be here also or am I missing something?  Also line 69

Reply:
Yes.